### DISCUSSION TITLE
Commands from caprover

### DISCUSSION THREAD

**Discussion by uhlhosting** [caprover/caprover#1245] [Q&A]
Is there a way to run commands from caprover in docker repositories? For example setting up a cron inside a running container or such. `"php /var/www/html/artisan schedule:run >> /dev/null 2>&1"`

**Comment by githubsaturn** [comment]
For one-off commands, you can simply just [shell into your container](https://caprover.com/docs/troubleshooting.html#how-to-run-shell-inside-my-application-inside-container)

For persistent changes, like cronjobs, you'll have to modify your Dockerfile ([example](https://stackoverflow.com/questions/37458287/how-to-run-a-cron-job-inside-a-docker-container))
